Disable the Python bindings for now. They could be in a slave port later if
needed, but I am not currently aware of anything that uses them.

--- src/CMakeLists.txt.orig	2020-12-28 00:28:41 UTC
+++ src/CMakeLists.txt
@@ -9,6 +9,3 @@ add_subdirectory(bankdata)
 add_subdirectory(include)
 add_subdirectory(lib)
 add_subdirectory(bin)
-if(Python_Development_FOUND)
-    add_subdirectory(python)
-endif()
